third_party/nvfuser/csrc/python_frontend/fusion_state.h

```diff
@@ -15,15 +16,16 @@ enum class StateType {
 };
 
 struct TORCH_CUDA_CU_API State {
-  State(size_t _index, StateType _stype) : index(_index), stype(_stype) {}
+  State(size_t _index, serde::StateType _stype)
```

Comment:
ah, StateType is generated in the header, so it might be tricky to keep that header as opt-in.

Reply:
We could have a function to map between the `flatbuffer` and `nvfuser` Enums, but I was trying to eliminate redundant code.